test/DynamoCoreTests/WorkspaceSaving.cs

```diff
@@ -522,7 +522,13 @@ public void CustomNodeSaveAsDoesGiveNewFunctionIdToNewCustomNode()
 
             Assert.AreNotEqual(Guid.Empty, initialId);
```

Comment:
This existing line `Assert.AreNotEqual(Guid.Empty, initialId);` will always be `true` ;)